pnpm-workspace.yaml

```
@@ -1,3 +1,4 @@
 packages:
   - "packages/*"
   - "examples/**"
+  - "!examples/cli"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)